tests/rhui3_tests/test_client_management.py

```diff
@@ -56,7 +93,56 @@ def test_05_create_cli_rpm():
     Expect.enter(connection, 'q')
     Expect.ping_pong(connection, "test -f /root/test_cli_rpm-3.0/build/RPMS/noarch/test_cli_rpm-3.0-1.noarch.rpm && echo SUCCESS", "[^ ]SUCCESS")
 
-def test_06_create_docker_cli_rpm():
+def test_08_remove_rhui_conf_rpm():
+    '''
+       remove rhui configuration rpm from client
+    '''
+    Util.remove_conf_rpm(cli)
```

Comment:
I wasn't aware of `remove_conf_rpm()`, but I looked it up in `util.py` and found this code:

```
        '''
        Remove RHUI configuration rpm from instance (which owns /etc/yum/pluginconf.d/rhui-lb.conf file)
        '''
        Expect.enter(connection, "")
        Expect.expect(connection, "root@")
        Expect.enter(connection, "([ ! -f /etc/yum/pluginconf.d/rhui-lb.conf ] && echo SUCCESS ) || (rpm -e `rpm -qf --queryformat %{NAME} /etc/yum/pluginconf.d/rhui-lb.conf` && echo SUCCESS)")
        Expect.expect(connection, "[^ ]SUCCESS.*root@", 60)
```
Hmm, that seems to have been written for RHUI 2. In RHUI 3, there's no such plugin. This method must be rewritten to check for `/etc/yum.repos.d/rh-cloud.repo` instead.